DMX.Core.Api.Tests.Unit/Services/Foundations/LabCommands/LabCommandServiceTests.cs

```diff
@@ -1,44 +1,47 @@
-﻿// ---------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// ---------------------------------------------------------------
-
-using System;
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
 using System.Linq.Expressions;
 using System.Runtime.Serialization;
 using DMX.Core.Api.Brokers.DateTimes;
-using DMX.Core.Api.Brokers.Loggings;
-using DMX.Core.Api.Brokers.Storages;
-using DMX.Core.Api.Models.Foundations.LabCommands;
-using DMX.Core.Api.Services.Foundations.LabCommands;
+using DMX.Core.Api.Brokers.Loggings;
+using DMX.Core.Api.Brokers.Storages;
+using DMX.Core.Api.Models.Foundations.LabCommands;
+using DMX.Core.Api.Services.Foundations.LabCommands;
 using Microsoft.Data.SqlClient;
-using Moq;
-using Tynamix.ObjectFiller;
+using Moq;
+using Tynamix.ObjectFiller;
 using Xeptions;
 using Xunit;
+using Xunit.Abstractions;
 
-namespace DMX.Core.Api.Tests.Unit.Services.Foundations.LabCommands
-{
-    public partial class LabCommandServiceTests
-    {
-        private readonly Mock<IStorageBroker> storageBrokerMock;
+namespace DMX.Core.Api.Tests.Unit.Services.Foundations.LabCommands
+{
+    public partial class LabCommandServiceTests
+    {
+        private readonly Mock<IStorageBroker> storageBrokerMock;
         private readonly Mock<IDateTimeBroker> dateTimeBrokerMock;
-        private readonly Mock<ILoggingBroker> loggingBrokerMock;
-        private readonly ILabCommandService labCommandService;
-
-        public LabCommandServiceTests()
-        {
-            this.storageBrokerMock = new Mock<IStorageBroker>();
-            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private readonly ILabCommandService labCommandService;
+
+        private readonly ITestOutputHelper output;
+
+        public LabCommandServiceTests(ITestOutputHelper output)
+        {
+            this.output = output;
+
```

Comment:
What's this? :)

Reply:
Debugging tool when the random time zones were melting our brains :) Accidentally committed, removing!